packages/mrgn-ui/src/components/action-box-v2/components/action-settings/components/bundle-tip.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { IconInfoCircle } from "@tabler/icons-react";
+
+import { cn } from "@mrgnlabs/mrgn-utils";
+
+import { Button } from "~/components/ui/button";
+import { Input } from "~/components/ui/input";
+import { Label } from "~/components/ui/label";
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "~/components/ui/tooltip";
+import { Form, FormControl, FormField, FormItem } from "~/components/ui/form";
+import { RadioGroup, RadioGroupItem } from "~/components/ui/radio-group";
+
+type BundleTipProps = {
+  toggleSettings: (mode: boolean) => void;
+  bundleTip: number;
+  changeBundleTip: (value: number) => void;
+};
+
+const priorityFeeOptions = [
+  {
+    label: "Normal",
+    value: 0,
+  },
+  {
+    label: "High",
+    value: 0.00005,
+  },
+  {
+    label: "Mamas",
+    value: 0.005,
```

Comment:
going from .00005 to .005 is a big jump no?

Reply:
unused code, has been removed ✅